docker/internal-auth/compose.plugins.yaml

```diff
@@ -24,6 +24,8 @@ services:
       - plugins-vol:/opt/plugins
 
   teams-app:
+    environment:
+      FEATURE_FLAG_ENABLE_INVITATIONS: false
```

Comment:
In internal mode, hard code disable by setting `FEATURE_FLAG_ENABLE_INVITATIONS` to `false`, even if user sets to true in `.env`.

Added to teams-app as it should be set here in addition to teams-api.